ui/selectors/selectors.js

```diff
@@ -3274,20 +3302,32 @@ export function getPermittedChains(state, origin) {
  * @param {object} state - The current state.
  * @returns {Array<string>} An empty array or an array of accounts.
  */
-export function getPermittedAccountsForCurrentTab(state) {
-  return getPermittedAccounts(state, getOriginOfCurrentTab(state));
+export function getPermittedEVMAccountsForCurrentTab(state) {
+  return getPermittedEVMAccounts(state, getOriginOfCurrentTab(state));
+}
+
+export function getPermittedEVMAccountsForSelectedTab(state, activeTab) {
+  return getPermittedEVMAccounts(state, activeTab);
+}
+
+export function getAllPermittedAccountsForCurrentTab(state) {
+  return getAllPermittedAccounts(state, getOriginOfCurrentTab(state));
 }
 
-export function getPermittedAccountsForSelectedTab(state, activeTab) {
-  return getPermittedAccounts(state, activeTab);
+export function getAllPermittedAccountsForSelectedTab(state, activeTab) {
+  return getAllPermittedAccounts(state, activeTab);
 }
 
-export function getPermittedChainsForCurrentTab(state) {
-  return getPermittedAccounts(state, getOriginOfCurrentTab(state));
+export function getPermittedEVMChainsForCurrentTab(state) {
```

Comment:
It looks like this function is being used as an input selector below. Is there a way we can make this a memoized selector so that we don't need to use `createDeepEqualSelector` below?

Reply:
This doesn't appear to be used at all anymore...

Reply:
I feel like we need to go through and understand where some of these EVM only selectors are still used...

For now I've removed this one since its not used anywhere: https://github.com/MetaMask/metamask-extension/pull/29887/commits/8c2973e3a2e21ea9b878b39c14e94c3fb80cdb6a